pkg/manager/member/pump_member_manager.go

```diff
@@ -440,21 +444,37 @@ func getPumpStartScript(tc *v1alpha1.TidbCluster) (string, error) {
 	if tc.Spec.EnableTLSCluster {
 		scheme = "https"
 	}
-	logLevel := tc.Spec.Pump.LogLevel
-	if logLevel == "" {
-		logLevel = "info"
-	}
 	err := pumpStartScriptTpl.Execute(buff, struct {
 		Scheme      string
 		ClusterName string
 		LogLevel    string
-	}{scheme, tc.Name, logLevel})
+	}{scheme, tc.Name, getPumpLogLevel(tc)})
 	if err != nil {
 		return "", err
 	}
 	return buff.String(), nil
 }
 
+func getPumpLogLevel(tc *v1alpha1.TidbCluster) string {
```

Comment:
Why not pass the pump config field directly? Passing a `TidbCluster` pointer around is both unnecessary and unsafe.

Reply:
Just wanted to hide the details about where the `pumpLogLevel` located in this function, this is a private function so I did not think too much, do you think it is necessary to change the function signature?

Reply:
OK, sgtm.